src/components/Charts/mutationsTree.js

```diff
@@ -0,0 +1,48 @@
+
+import React from 'react';
+import Container, {Title} from "./styles";
+
+class MutationsTree extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
+
+    createVariants(variants, level){
+        let variantLines=[];
+        for(const variant of variants){
```

Comment:
This can be rewritten in a more functional way like this
```
return variants.flatMap((variant) => [
   `${'---'.repeat(level)}variant: ${variant.name} (${mutations})`,
   ...createVariants(variant.subs, level+1)
])
```